Core packages in OSS (#1351)
ezsystems/developer-documentation: docs/guide/bundles.md

## Core packages

!!! tip

>     The core packages are included in Ibexa Open Source.

Reviewer: Howabout using the active voice?
"Ibexa Open Source includes the core packages."?

Author: I've tried to make it clear that core packages are the *only* ones in OSS, without actually saying "only" :)

Reviewer: In this case, I like "Ibexa Open Source is composed of the core packages" best, despite its passivevoiciness

>     The core packages are included in Ibexa Open Source.

Author: ```suggestion
    Ibexa Open Source is composed of the core packages.
```
